addons/vscode/src/extension.ts

```diff
@@ -122,6 +122,18 @@ export async function getCliPath(extensionPath?: string): Promise<string> {
 	return (state.resolved = await resolvePath());
 }
 
+function getCliInputArgs(inputs?: { [key: string]: string }): string[] {
+	return Object.entries(inputs ?? {})
+		.filter(([k, _]) => k.trim() !== "")
+		.map(([k, v]) => ["--input", `${k}=${vscodeVariables(v)}`])
```

Comment:
maybe we need somehow quote or escape the `v` here?

Reply:
Not very familiar with NodeJS, but doesn't the [spawn](https://nodejs.org/api/child_process.html#child_processspawncommand-args-options) handle escaping of command-line arguments?

Reply:
good catch

Reply:
Should we apply `vscodeVariables` here? What's the use case of substituting vscode variables here.

Reply:
I don't have a specific use case myself, but I can imagine someone might want to use `workspaceFolderBasename` as a project name (or project ID), perhaps?

I'm not sure, removing this feature wouldn't affect me personally.

Reply:
perhaps we could just remove it to keep same behaviors as all other typst services. We can add it back once we consider it necessary in future.

Reply:
OK, I can remove it in #299.